modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ObjectDefinitionNode/ObjectDefinitionNode.tsx

```diff
@@ -65,14 +42,7 @@ export function ObjectDefinitionNode({
 		false
 	);
```

Comment:
Hey @Artartur, we need to stop to use this state. Since we already create the propertie in context to replace that.